index.js

```diff
@@ -180,16 +180,18 @@ class ServerlessWSGI {
     }
 
     const port = this.options.port || 5000;
+    const host = this.options.host || 'localhost';
 
     return new BbPromise((resolve, reject) => {
       var status = child_process.spawnSync(this.pythonBin, [
         path.resolve(__dirname, 'serve.py'),
         this.serverless.config.servicePath,
         this.wsgiApp,
-        port
+        port,
+        host
       ], { stdio: 'inherit' });
       if (status.error) {
-        reject(status.error);
+        reject(status.stdout.toString());
```

Comment:
On line 192 we have `{ stdio: 'inherit' }`, so I'm curious about this change. Did you have any problems with the previous implementation? From my testing, `status.stdout` may be null and throws when `toString()` is called, and the output is already emitted to the terminal due to the `inherit` setting anyway.

Reply:
Hey, please don't mind my rudimentary javascript skills.

I was having issues on specifying an alias to the --host parameter, for some reason I couldn't understand the command was failing and returning the help each time I was running it with the previous alias (-h).

I've changed this line when I was troubleshooting the reason and forgot to switch back to the original one. 

BTW I haven't  noticed the `{ stdio: 'inherit' }` so now I feel even stupider for added the status.stdout.toString() 😕 

Bottom line, we should change that to the original value.

Reply:
👍 No problem, thanks for the clarification - I'll update it before merging